packages/website-backend/test/unit/api/controllers/RealTimeReportsController.spec.ts

```diff
@@ -0,0 +1,366 @@
+import supertest from 'supertest';
+import RealTimeReportsController from '../../../../src/api/RealTimeReportsController.js';
+import { PlatformTest } from '@tsed/common';
+import Server from '../../../../src/Server.js';
+import sinon from 'sinon';
+import {
+  MutationTestingReportService,
+  Project,
+  ProjectMapper,
+  RealTimeMutantsBlobService,
+} from '@stryker-mutator/dashboard-data-access';
+import {
+  DataAccessMock,
+  MutationEventResponseOrchestratorMock,
+} from '../../../helpers/TestServer.js';
+import DataAccess from '../../../../src/services/DataAccess.js';
+import { expect } from 'chai';
+import MutationEventResponseOrchestrator from '../../../../src/services/real-time/MutationEventResponseOrchestrator.js';
+import { MutationEventResponseHandler } from '../../../../src/services/real-time/MutationEventResponseHandler.js';
+import utils from '../../../../src/utils.js';
+import { MutantStatus } from 'mutation-testing-report-schema';
+import { createMutationTestResult } from '../../../helpers/mutants.js';
+
+describe(RealTimeReportsController.name, () => {
+  const apiKey = '1346';
+
+  let request: supertest.SuperTest<supertest.Test>;
+  let findReportStub: sinon.SinonStubbedMember<
+    MutationTestingReportService['findOne']
+  >;
+  let findProjectStub: sinon.SinonStubbedMember<ProjectMapper['findOne']>;
+  let removeResponseHandlerStub: sinon.SinonStubbedMember<
+    MutationEventResponseOrchestrator['removeResponseHandlers']
+  >;
+  let responseHandlerForProjectStub: sinon.SinonStubbedMember<
+    MutationEventResponseOrchestrator['createOrGetResponseHandler']
+  >;
+  let getEventsStub: sinon.SinonStubbedMember<
+    RealTimeMutantsBlobService['getReport']
+  >;
+  let createBlobStub: sinon.SinonStubbedMember<
+    RealTimeMutantsBlobService['createReport']
+  >;
+  let saveReportStub: sinon.SinonStubbedMember<
+    MutationTestingReportService['saveReport']
+  >;
+  let realTimeDeleteBlobStub: sinon.SinonStubbedMember<
+    RealTimeMutantsBlobService['delete']
+  >;
+  let deleteBlobStub: sinon.SinonStubbedMember<
+    MutationTestingReportService['delete']
+  >;
+  let project: Project;
+
+  beforeEach(async () => {
+    await PlatformTest.bootstrap(Server)();
+    request = supertest(PlatformTest.callback());
+
+    const dataAccess = PlatformTest.get<DataAccessMock>(DataAccess);
```

Comment:
Why not use the `dataAccess` directly? Instead of copying over all values to separate `*Stub` variables? Seems like a lot of code for little gain.